lib/TPP/ConvertLinalgToXsmm.cpp

```diff
@@ -101,17 +102,16 @@ static void replaceOpWithUnary(RewriterBase &rewriter,
       loc, integer64, kind, dims, flags, dtype);
   SmallVector<Value> invokeOperands;
   invokeOperands.push_back(dispatched);
-  invokeOperands.append(linalgOp->getOperands().begin(),
-                        linalgOp->getOperands().end());
+  invokeOperands.append(operands.begin(), operands.end());
```

Comment:
Does any test capture this change?

Reply:
yes, relu_5 and add_9. The add has 2 operands and the xsmm lowering has 3 operands. Same for the relu 1 operand and 2 operands in xsmm.